sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessMessageEventArgs.cs

```diff
@@ -26,6 +26,16 @@ public class ProcessMessageEventArgs : EventArgs
         /// </summary>
         public CancellationToken CancellationToken { get; }
 
+        /// <summary>
+        /// The entity path corresponding to the entity that the message was received from.
+        /// </summary>
+        public string EntityPath => _receiver.EntityPath;
+
+        /// <summary>
+        /// The fully qualified namespace of the Service Bus namespace that the message was received from.
```

Comment:
```suggestion
        /// The fully qualified Service Bus namespace that the message was received from.
```

This read a bit awkwardly to me; I used the [Receiver comment](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs#L28) as the basis for the rewording.